### ISSUE TITLE
The instructions on your websites are not working...

### ISSUE THREAD

**Issue by tobiashochguertel** [hook0/hook0#5]
https://documentation.hook0.com/docs/docker-compose

```
❯ docker compose -f self-hosted/docker/docker-compose.yaml up --build --detach
Failed to load /opt/docker-volumes/others/hook0/hook0/self-hosted/docker/api.env: open /opt/docker-volumes/others/hook0/hook0/self-hosted/docker/api.env: no such file or directory
```

and the instructions for the bare-metal installations are hard to read:
https://documentation.hook0.com/docs/manual-setup

![CleanShot 2023-09-11 at 21 41 24](https://github.com/hook0/hook0/assets/3332669/124ab685-7b75-4591-9fbf-7b6621ddae04)

**Comment by dsferruzza** [comment]
Hello!

There was indeed an issue in our Docker Compose file; I just fixed it! (see 370dd6d514c5ba7935a347c39ce68977664b6ccc)
Thanks for the report!

About the hard to read instructions, every seems fine to me as of now; I suspect a temporary issue with our documentation website's provider.
Does it work for you too now?

Let me know if you need something else!

**Comment by tobiashochguertel** [comment]
@dsferruzza now it looks nice 👍 
![image](https://github.com/hook0/hook0/assets/3332669/25521d47-1a8e-4eac-8443-557b101c4f72)

Thanks!
